### PR TITLE
Update obsolete blockIndex

### PR DESCRIPTION
**Pull request by ipdae**

### CODE REVIEW

File: Lib9c/Action/SellCancellation7.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionObsolete(2000000)]
+    [ActionObsolete(2100000)]
```
Comment (boscohyun) [inline Lib9c/Action/SellCancellation7.cs:21]: This action should contains below in the `Execute()` method.
```
CheckObsolete(2100000, context);
```

File: Lib9c/Action/Sell7.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionObsolete(2000000)]
+    [ActionObsolete(2100000)]
```
Comment (boscohyun) [inline Lib9c/Action/Sell7.cs:20]: This action should contains below in the `Execute()` method.
```
CheckObsolete(2100000, context);
```

File: Lib9c/Action/MonsterCollect2.cs [hunk 3]
```diff
@@ -12,7 +12,7 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionObsolete(2000000)]
+    [ActionObsolete(2100000)]
```
Comment (boscohyun) [inline Lib9c/Action/MonsterCollect2.cs:15]: This action should contains below in the `Execute()` method.
```
CheckObsolete(2100000, context);
```

File: Lib9c/Action/MigrationLegacyShop0.cs [hunk 4]
```diff
@@ -10,7 +10,7 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionObsolete(2000000)]
+    [ActionObsolete(2100000)]
```
Comment (boscohyun) [inline Lib9c/Action/MigrationLegacyShop0.cs:13]: This action should contains below in the `Execute()` method.
```
CheckObsolete(2100000, context);
```

File: Lib9c/Action/HackAndSlash6.cs [hunk 5]
```diff
@@ -16,7 +16,7 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionObsolete(2000000)]
+    [ActionObsolete(2100000)]
```
Comment (boscohyun) [inline Lib9c/Action/HackAndSlash6.cs:19]: This action should contains below in the `Execute()` method.
```
CheckObsolete(2100000, context);
```

File: Lib9c/Action/CancelMonsterCollect.cs [hunk 6]
```diff
@@ -12,7 +12,7 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionObsolete(2000000)]
+    [ActionObsolete(2100000)]
```
Comment (boscohyun) [inline Lib9c/Action/CancelMonsterCollect.cs:15]: This action should contains below in the `Execute()` method.
```
CheckObsolete(2100000, context);
```

**Review by boscohyun — APPROVED** [review]

**Review by area363 — APPROVED** [review]

**Review by longfin — APPROVED** [review]

**Pull request merged**